document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/transaction/TransactionManager.java

```diff
@@ -87,7 +87,7 @@ private static RestTemplate createRestTemplate() {
      * @return the private endpoint transaction url with the transaction id
      */
     private String getBaseUrl(String id) {
-        return new StringBuilder("/private/transactions/").append(id).toString();
+        return new StringBuilder("/private").append(id).toString();
```

Comment:
This is a bit misleading. "/transaction/[ID]" is not an id.

Reply:
altered